client/src/main/proto/spine/client/entities.proto

```diff
@@ -73,8 +74,26 @@ message EntityFilters {
     // Match the entities by the value of the Entity Columns.
     map<string, google.protobuf.Any> column_filter = 2;
 
+    // Match the entities which are created later then the given time.
```

Comment:
I'll leave it `created_after` for this time. In future, this field is to be eliminated and some more convenient `repeated` field for parameter comparison is to be introduced.